src/test/objectstore/store_test.cc

```diff
@@ -295,6 +296,26 @@ TEST_P(StoreTest, FiemapHoles) {
     ASSERT_TRUE((m.size() == 1 &&
 		 m[0] > SKIP_STEP * (MAX_EXTENTS - 1)) ||
 		 (m.size() == MAX_EXTENTS && extents_exist));
+
+    // fiemap test from SKIP_STEP to SKIP_STEP * (MAX_EXTENTS - 2) + 3
+    // reset bufferlist and map
+    bl.clear();
+    m.clear();
+    e.clear();
+    store->fiemap(cid, oid, SKIP_STEP, SKIP_STEP * (MAX_EXTENTS - 2) + 3, bl);
+    p = bl.begin();
+    ::decode(m, p);
+    cout << " got " << m << std::endl;
+    ASSERT_TRUE(!m.empty());
+    ASSERT_GE(m[SKIP_STEP], 3u);
```

Comment:
I see.  because kstore fiemap always return [0, o->onode.size]  regardless of  the offset value.

@liewegas , hi, sage.  is this break the fiemap logic for kstore?  when fiemap(offset=1024, 2048) is called,  for other store bacnkend, it will return extents begin with m[1024] = xxx. while kstore is permanently returning m[0] = onode.size here. 
So shall we unified the kstore logic for fiemap or skip this assertion here?